File: kattis/python/takkar.py
def main():
    trump_button = int(input())
    kim_button = int(input())

    if trump_button > kim_button:
        print("MAGA!")
    elif trump_button < kim_button:
        print("FAKE NEWS!")
    else:
        print("WORLD WAR 3!")

if __name__ == "__main__":
    main()

File: kattis/haskell/takkar.hs
compareButtons :: Int -> Int -> String
compareButtons trump kim
  | trump > kim = "MAGA!"
  | trump < kim = "FAKE NEWS!"
  | otherwise   = "WORLD WAR 3!"

main :: IO ()
main = do
  trumpLine <- getLine
  kimLine <- getLine
  let trump = read trumpLine :: Int
      kim = read kimLine :: Int
  putStrLn (compareButtons trump kim)
